src/components/ForgotPassword.js

```diff
@@ -0,0 +1,54 @@
+// react libraries
+import React, { Component } from 'react';
+
+// Moduler importations
+import Logo from './Logo';
+import '../styles/_form.scss';
+/**
+ * @desc renders home page
+ */
+class ForgotPassword extends Component {
```

Comment:
#### Nitpick
Why not make this a functional stateless component.

Reply:
Done